addons/web/tests/test_perf_load_menu.py

```diff
@@ -18,8 +18,8 @@ def test_performance_session_info(self):
         self.authenticate(user.login, "info")
 
         self.env.registry.clear_all_caches()
-        # cold ormcache (only web: 37, all module: 117)
-        with self.assertQueryCount(117):
+        # cold ormcache (only web: 37, all module: 118)
+        with self.assertQueryCount(118):
```

Comment:
related to https://github.com/odoo/enterprise/pull/77895/files#diff-03f8d2bd889c06596ed463a837be47a5cef572cb57977fbf76dca08892a57e8eR11 

where we add another xml reference that will used in a new call of `_xmlid_to_res_id()`. See https://github.com/odoo/enterprise/blob/master/account_accountant/models/ir_ui_menu.py#L18